### ISSUE TITLE
Fail to build with Go 1.11

### ISSUE THREAD

**Issue by infertux** [etix/mirrorbits#84]
```bash
% go build
../../garyburd/redigo@v2.0.0+incompatible/redis/pool.go:28:2: use of internal package github.com/gomodule/redigo/internal not allowed
% go version
go version go1.11.1 linux/amd64
```

**Comment by etix** [comment]
Did you use the Makefile? I know it's not perfect, but that's the only way to get the current git revision into the binary.

```
% cd $GOPATH/src/github.com/etix/mirrorbits
% make
% bin/mirrorbits -D
```

**Comment by infertux** [comment]
The issue was actually about the new [module feature](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) that came with Go 1.11. I had the env variable `GO111MODULE=on` set from another Go project and this changes the behavior of `go get` so when I ran `go get -u github.com/etix/mirrorbits` it set up the repo using modules and `make` failed running the `go build` command.

I tried to replicate the issue in Travis [here](https://travis-ci.org/infertux/mirrorbits/builds/451311469) but it's not possible because Travis clones the repo using `git clone` (not `go get`) therefore it doesn't care about the `GO111MODULE` variable.
